cmd/minishift/cmd/image/export.go

```diff
@@ -144,5 +145,6 @@ func createLogFile() *os.File {
 func init() {
 	imageExportCmd.Flags().BoolVar(&exportAll, "all", false, "Exports all images currently available in the Docker daemon.")
 	imageExportCmd.Flags().BoolVar(&logToFile, "log-to-file", false, "Logs export progress to file instead of standard out.")
+	imageExportCmd.Flags().BoolVar(&overwrite, "overwrite", false, "Exports  images currently available in the Docker daemon.")
```

Comment:
`--overwrite     Exports  images currently available in the Docker daemon` does not communicate its purpose.  I would suggest `--overwrite     Forces an image exports images  when the image is already available in the Docker daemon`.